conf/script/v2/subprocess_wrapper.py

```diff
@@ -0,0 +1,13 @@
+import subprocess
+
+class CmdException(Exception):
```

Comment:
We already have this in cortx-ha/ha/execute.py

Please create subclass in file if execute.py is not providing all requirement

Reply:
Switched to SimpleCommand class usage. Thanks for pointing.